src/main/java/com/encora/Tree.java

```diff
@@ -0,0 +1,10 @@
+package com.encora;
+
+public interface Tree {
+
+    Node add(int value);
+    boolean contains(int value);
+    Node delete(int value);
+    String printInOrder();
```

Comment:
Nitpick: we can avoid adding this method and rely on `toString()` for the actual class.